glutton.go

```diff
@@ -131,7 +131,7 @@ func (g *Glutton) makeID() error {
 		return err
 	}
 	if f, err := os.OpenFile(filePath, os.O_RDWR, 0644); os.IsNotExist(err) {
-		g.id = uuid.NewV4()
+		g.id, _ = uuid.NewV4()
```

Comment:
You should handle the error here.
Why is this working if we set the uuid version to [1.2.0](https://github.com/mushorg/glutton/blob/master/go.mod#L22)?

Reply:
That's a good question. I'll do some more testing and see if I can figure out why my build didn't pull the right version of uuid.